resources/locale.json

```diff
@@ -745,195 +745,195 @@
+        "UI_PEACOCK_TRADITIONS_STALKER_HUD": "Stelle den Stalker ruhig und verstecke ihn",
+        "UI_PEACOCK_TRADITIONS_MILITIA": "Die Miliz",
+        "UI_PEACOCK_TRADITIONS_MILITIA1_DESC": "Er ist nicht alleine... Eliminiere alle Miliz Soldaten in der Medow Brown Lounge um den Bereich zu säubern.",
+        "UI_PEACOCK_TRADITIONS_MILITIA1_HUD": "Eliminiere alle Miliz Soldaten in der Medow Brown Lounge",
+        "UI_PEACOCK_TRADITIONS_MILITIA2_HUD": "Eliminiere alle Miliz Soldaten im Lieferbereich",
+        "UI_PEACOCK_TRADITIONS_DETONATOR": "Besorge den Zünder",
+        "UI_PEACOCK_TRADITIONS_DETONATOR_DESC": "Der Stalker hat geredet... Ein Mann verkleidet als hochrangiger Sicherheitsdienst im Keller hat den Zünder, besorge ihn jetzt!",
+        "UI_PEACOCK_TRADITIONS_PLANB_DESC": "47, Julian hatte einen Plan B: Er hat einen tödlichen Virus in der Küche freigelassen! Zwei Leute sind infiziert. Eliminiere die zwei Infizierten und verstecke sie im Kühlschrank. Wir holen die Leichen später ab.",
+        "UI_PEACOCK_TRADITIONS_PLANB_HUD": "Eliminiere die Infizierten und stecke sie in den Kühlschrank",
+        "UI_PEACOCK_TRADITIONS_ANTIVIRUS": "Das Antivirus",
+        "UI_PEACOCK_TRADITIONS_ANTIVIRUS_HUD": "Injiziere dich mit dem Antivirus, JETZT!",
+        "UI_PEACOCK_TRADITIONS_ERASE": "Beseitige deine Spuren",
+        "UI_PEACOCK_TRADITIONS_ERASE_DESC": "Nimm die Bänder der Überwachungskamera mit. Wir brauchen sie intakt.",
+        "UI_PEACOCK_TRADITIONS_ERASE_HUD": "Nimm die Bänder der Überwachungskamera mit.",
+        "UI_PEACOCK_TRADITIONS_JACKIE_HUD": "Eliminiere Jackie Carrington",
+        "UI_PEACOCK_TRADITIONS_ACCOMPLICE_HUD": "Eliminiere den Komplizen",
+        "UI_PEACOCK_LEEHONG": "Das Lee Hong Attentat",
+        "UI_PEACOCK_LEEHONG_DESC": "Guten Abend 47. Unser Ziel ist Lee Hong, der Anführer der Red Dragon Triade und Angus Pritchard, ein reicher Antiquitätensammler aber auch eine ehemalige FBI Quelle.Beide sollen sich in ein paar Minuten in einem Fisch Restraurant Treffen um ein goldenes Idol für Informationen über unseren Klienten zu tauschen.Der Klient möchte das du diese Informationen besorgst.Hong und Pritchard möchten nicht zusammen gesehen werden also essen sie an getrennten Tischen, aber sie werden den Austausch in ungefähr 10 Minuten durchführen.Das darf nicht passieren, 47.Unser Klient möchte das sie im Glauben gelassen werden, das der Tausch zwischen den Beiden schiefging um keine Aufmerksamkeit zu erregen.Sei kreativ.Ich habe gehört die Triade benutzt spezielle Munition für ihre Gewehre.Vielleicht kannst du diese Informationen nutzen um die Triade mit Pritchards Mord zu beschuldigen...\nDie Uhr tickt 47. Viel Glück.\nGemacht von einem Peacock Community Mitglied bekannt als kercyx.",
```

Comment:
Also it's "Fischrestaurant". Also you didn't add a space after the end of a sentence multiple times here.